app/templates/play/menu/options-view.pug

```diff
@@ -40,6 +40,14 @@
       span(data-i18n="options.editor_config_behaviors_label") Enable Smart Behaviors
     span.help-block(data-i18n="options.editor_config_behaviors_description") Autocompletes brackets, braces, and quotes.
 
+  .form-group.checkbox
+    label(for="option-wide-editor")
+      input#option-wide-editor(name="wide-editor", type="checkbox", checked=aceConfig.preferWideEditor)
+      span.custom-checkbox
+        .glyphicon.glyphicon-ok
+      span(data-i18n="options.editor_config_wide_editor_label") Enable Wide Editor
```

Comment:
why have we added strings even though we have their i18n versions?

Reply:
like a backup. but it is not needed ya